src/org/company/LargeClass.java

```diff
@@ -0,0 +1,25 @@
+package org.company;
+
+/**
+ * @since Current.Project.Version
+ */
+public class LargeClass
+{
+    
```

Comment:
This line is not needed. and then for everything below, there should only be one space between things for the long lines. EX:
```
private final String  fieldOne    = "field one value";
private         String  fieldTwo    = "field two value";
private final Object fieldThree = new Object();
```